examples/td3/config.yaml

```diff
@@ -1,21 +1,15 @@
 # Environment
 env_name: HalfCheetah-v4
-env_task: ""
 exp_name: "debugging"
-env_library: gym
-record_video: 0
-normalize_rewards_online: 0
-normalize_rewards_online_scale: 5
-normalize_rewards_online_decay: 0.99
 total_frames: 1000000
-frames_per_batch: 1000
+frames_per_batch: 200
 max_frames_per_traj: 1000
 frame_skip: 1
 from_pixels: 0
 seed: 0
 
 # Collection
-init_random_frames: 25000
```

Comment:
Here also not big issue but exact paper value would be 10k

Reply:
Got it